docs/ossm/create-mesh/README.md

```diff
@@ -0,0 +1,201 @@
+
+> **_NOTE:_** `discoverySelectors` is not a security boundary. Istiod will continue to have access to all namespaces even when you have configured your `discoverySelectors`.
+
+#### DiscoverySelectors vs. Sidecar
+The `discoverySelectors` configuration enables users to dynamically restrict the set of namespaces that are part of the mesh. A `Sidecar` resource also controls the visibility of sidecar configurations and what gets pushed to the sidecar proxy. What are the differences between them?
+
+- The `discoverySelectors` configuration declares what Istio control plane watches and processes. Without `discoverySelectors` configuration, the Istio control plane watches and processes all namespaces/services/endpoints/pods in the cluster regardless of the sidecar resources you have.
+- `discoverySelectors` is configured globally for the mesh by the mesh administrators. While `Sidecar` resources can also be configured for the mesh globally by the mesh administrators in the MeshConfig root namespace, they are commonly configured by service owners for their namespaces.
+
+You can use `discoverySelectors` with `Sidecar` resources. You can use `discoverySelectors` to configure at the mesh-wide level what namespaces the Istio control plane should watch and process. For these namespaces in the Istio service mesh, you can create `Sidecar` resources globally or per namespace to further control what gets pushed to the sidecar proxies.
+
+ #### Using DiscoverySelectors
+ `discoverySelectors` field accepts an array of Kubernetes [selectors](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#resources-that-support-set-based-requirements). The exact type is `[]LabelSelector`, as defined [here](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#resources-that-support-set-based-requirements), allowing both simple selectors and set-based selectors. These selectors apply to labels on namespaces.
+
+You can configure each label selector for expressing a variety of use cases, including but not limited to:
+
+- Arbitrary label names/values, for example, all namespaces with label `istio-discovery=enabled`
+- A list of namespace labels using set-based selectors which carries OR semantics, for example, all namespaces with label `istio-discovery=enabled` OR `region=us-east1`
+- Inclusion and/or exclusion of namespaces, for example, all namespaces with label `istio-discovery=enabled` AND label key `app` equal to `helloworld`
+
+Example of Istio CR with `discoverySelectors` defined:
+```yaml
+kind: Istio
+apiVersion: sailoperator.io/v1alpha1
+metadata:
+  name: ossm3
+spec:
+  namespace: istio-system3
+  values:
+    meshConfig:
+      discoverySelectors:
+        - matchExpressions:
+          - key: maistra.io/member-of
+            operator: DoesNotExist
+  updateStrategy:
+    type: InPlace
+  version: v1.23.0
+```
+
+## Installing the Sidecar
```

Comment:
After reading this, I think the topic of installing the sidecar ("Adding services to a mesh" seems more general) should be broken out into a separate page, similar to how the [OSSM 2 docs](https://docs.openshift.com/container-platform/4.17/service_mesh/v2x/prepare-to-deploy-applications-ossm.html).

Reply:
Moved the sidecar content to separate page and added references.